README.md

```diff
@@ -492,12 +492,13 @@ commit [0360811][].  It was created via `git clone --bare` and `git push
 [bats-orig]: https://github.com/sstephenson/bats
 [0360811]: https://github.com/sstephenson/bats/commit/03608115df2071fff4eaaff1605768c275e5f81f
 
-This [bats-core repo](https://github.com/bats-core/bats-core) is now the official Bats project.
+This [bats-core repo](https://github.com/bats-core/bats-core) is the community-maintained Bats project.
 
 ### Why was this fork created?
 
-The original Bats repository is no longer maintained and write access to it could not be obtained. This fork allowed ongoing maintenance and forward progress for Bats.
+There was an initial [call for maintainers][call-maintain] for the original Bats repository, but write access to it could not be obtained. As the main repository appeared inactive, this fork allowed ongoing maintenance and forward progress for Bats.
```

Comment:
Thanks for this. I didn't realize the `call-maintain` link wasn't in this section. 

I like the general wording. The only thing is that I think people browsing on GitHub might conflate the _code repository_ with the GitHub repository as a whole, so they might get the opposite perception that the GitHub repo is inactive (which is not true of course, there are a lot of PRs and discussion).

Reply:
Meh. with both the link to the original call for maintainers and the link to the issue documenting the fork, those interested can derive the time allowed.

But I agree with the repo phrasing. will tweak a bit more.